controller:
  ros__parameters:
    broadcastTime: 1
    commitmentUpdateTime: 5
    pPerceiveLightSources: 0.1
    numberOfTargets: 4
    colorsOfTargets: ["yellow", "green", "magenta", "brown"]
    fov: 45.0
    commsType: 0
    hardTurnOnAngleThreshold: 50.0
    softTurnOnAngleThreshold: 20.5
    noTurnOnAngleThreshold: 10.0
    maxSpeed: 10.0
    logDirectoryName: "/mnt"
    experimentName: "4_targets_collisions_10_perception_50_ht"
